segment_data/strava_segment_data_load.py

```diff
@@ -0,0 +1,20 @@
+import requests
+import json
+
+ACCESS_TOKEN = 'ae6026e2583caaf9348db334197e5c9484c751b3'
```

Comment:
Remove access token from codebase.  I'd put it in a separate config file or something like that.